Sources/Plasma/FeatureLib/pfLocalizationMgr/pfLocalizationDataMgr.cpp

```diff
@@ -169,39 +182,34 @@ XML_Memory_Handling_Suite gHeapAllocator = {
 
 void XMLCALL LocalizationXMLFile::StartTag(void *userData, const XML_Char *element, const XML_Char **attributes)
 {
-    ST::string wElement = element;
     LocalizationXMLFile *file = (LocalizationXMLFile*)userData;
-    std::map<ST::string, ST::string> wAttributes;
-
-    for (int i = 0; attributes[i]; i += 2)
-        wAttributes[attributes[i]] = attributes[i+1];
-
-    LocalizationXMLFile::tagInfo parentTag;
-    if (!file->fTagStack.empty())
-        parentTag = file->fTagStack.top();
 
-    LocalizationXMLFile::tagInfo newTag;
-    newTag.fTag = wElement;
-    newTag.fAttributes = wAttributes;
+    // Need a dummy parent tag on the stack at minimum
+    if (file->fTagStack.empty())
+        file->fTagStack.emplace();
```

Comment:
This initial tag emplacement could be moved to `LocalizationXMLFile::Parse` instead of checking it on every `StartTag`